CMakeLists.txt

```diff
@@ -20,6 +20,7 @@ find_package(
 	COMPONENTS Interpreter Development.Module NumPy
 	REQUIRED)
 
+include_directories(${PYTHON_INCLUDE_DIRS})
```

Comment:
Never use the old-style folder-based `include_directories` and friends! Link to `Python::Module` instead, that will also add the includes needed.

Reply:
Is it because SWIG needs the include to make the file? In that case, you could set SWIG_USE_TARGET_INCLUDE_DIRECTORIES on the target: https://cmake.org/cmake/help/latest/module/UseSWIG.html#properties-on-targets

Reply:
Excluding line 23 linked here leads to a `Python.h not found` error

Reply:
This looks like it's failing on compiling, not swigging - but let's try this define anyway.